scripts/qr-generator.js

```diff
@@ -22,10 +22,10 @@ if (!SECRET_KEY || !BASE_URL) {
 const amount = process.argv[2] ? parseInt(process.argv[2]) : 1;
 
 for (let i = 1; i <= amount; i++) {
-  const userId = uuidv4();
-  const payload = { userId };
+  const user_id = uuidv4();
+  const payload = { user_id };
   const token = jwt.sign(payload, SECRET_KEY);
-  const qr_png = qr.image(BASE_URL+"?" + token, {
+  const qr_png = qr.image(BASE_URL+"/welcome?token=" + token, {
```

Comment:
I have changed the url to `/welcome?token=sometoken` according to figma naming
or should we go for `/auth?token=sometoken`

Reply:
I think I prefer `/auth?token=sometoken`

Reply:
done🫡